src/main/java/org/gridsuite/filter/server/entities/LineFilterEntity.java

```diff
@@ -0,0 +1,47 @@
+package org.gridsuite.filter.server.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import javax.persistence.*;
+
+import java.util.Set;
+
+@Getter
+@NoArgsConstructor
+@SuperBuilder
+@Entity
+@Table(name = "line_filter")
+public class LineFilterEntity extends AbstractGenericFilterEntity {
+    @Column(name = "countries1")
+    @ElementCollection
+    @CollectionTable(foreignKey = @ForeignKey(name = "lineFilterEntity_countries_fk1"), indexes = {@Index(name = "lineFilterEntity_countries_idx1", columnList = "lineFilterEntity_name")})
+    Set<String> countries1;
+
+    @Column(name = "countries2")
+    @ElementCollection
+    @CollectionTable(foreignKey = @ForeignKey(name = "lineFilterEntity_countries_fk2"), indexes = {@Index(name = "lineFilterEntity_countries_idx2", columnList = "lineFilterEntity_name")})
+    Set<String> countries2;
+
+    @Column(name = "substationName1")
+    String substationName1;
+    @Column(name = "substationName2")
+    String substationName2;
+
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
avoid using EAGER (LAZY instead)

Reply:
what use is lazy if we always fetch it when we get the filter ?

Reply:
In getFiltersNames() method, we don't need to fetch nominalVoltage1 and nominalVoltage2 field. 
It's the same for countries1 and countries2 field, and these fields are handled lazily by default.

Reply:
i've forgotten to write the custom query for getFiltersNames(), it should be added in my next push